### PR TITLE
Delete User Portal

### PR DESCRIPTION
**Pull request by aimansharif**
Ready for review

### CODE REVIEW

File: src/main/resources/Javascript/Register_Script.js [hunk 1]
```diff
@@ -25,6 +27,27 @@ function myFunction() {
     else{
         alert("Password must adhere to guidelines")
     }
+=======
+=======
+>>>>>>> 35-delete-user-portal
```
Comment (nazifatanzim) [inline src/main/resources/Javascript/Register_Script.js:32]: Please remove the leftover merge conflict messages

File: src/main/resources/Javascript/Register_Script.js [hunk 2]
```diff
@@ -25,6 +27,27 @@ function myFunction() {
...
+        success: function (data) {
+            document.getElementById("userHidden").value = document.getElementById("username").value
+            document.getElementById("passHidden").value = document.getElementById("password").value
+            createFunction()
+        },
+        error: function (data) {
+            alert("Username already in use");
+        }
+    });
+<<<<<<< HEAD
```
Comment (nazifatanzim) [inline src/main/resources/Javascript/Register_Script.js:47]: here too

File: src/test/java/SYSC6/Project/UserControllerTest.java [hunk 3]
```diff
@@ -104,8 +104,8 @@ void list() throws Exception {
         JSONObject obj = new JSONObject();
         String jsonStringTexpected = "{\"data\":[],\"recordsFiltered\":5,\"recordsTotal\":5,\"draw\":0}";
         this.mockMvc.perform(post("/rest/api/usertable")
-                .contentType(MediaType.APPLICATION_JSON)
-                .content(String.valueOf(obj)))
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(String.valueOf(obj)))
                 .andExpect(status().is2xxSuccessful())
```
Comment (nazifatanzim) [inline src/test/java/SYSC6/Project/UserControllerTest.java:109]: unnecessary whitespace. please change this to avoid unnecessary merge conflicts for others

**Review by nazifatanzim — CHANGES_REQUESTED** [review]

**Comment by nazifatanzim** [comment]
Tests and code look good. Please address the changes I've requested. :)

**Review by NicolasDuciaume — APPROVED** [review]
After verification of every file affected I can confirm that the addition of your functioning code will not impact any of the other feature and should merge safely

File: src/main/java/SYSC6/Project/Main_Controller.java [hunk 4]
```diff
@@ -399,7 +399,6 @@ public Admin getAdmin(Long id){
         return users;
     }
 
-
```
Comment (nazifatanzim) [inline src/main/java/SYSC6/Project/Main_Controller.java]: Good thing this merge conflict was resolved!!!

**Review by nazifatanzim — APPROVED** [review]
